pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -476,9 +477,16 @@ func (rs *ReplsetSpec) setSafeDefauts(log logr.Logger) {
 
 	// Replset size can't be 0 or 1.
 	// But 2 + the Arbiter is possible.
-	if rs.Size < 2 {
-		loginfo(fmt.Sprintf("Replset size will be changed from %d to %d due to safe config", rs.Size, defaultMongodSize))
-		rs.Size = defaultMongodSize
+	if rs.ClusterRole == ClusterRoleConfigSvr {
```

Comment:
but what if Arbiter is enabled?
in such case, we receive 3 members and 1 arbiter, as result we receive high probability to get split-brain

Reply:
Should I change `minSafeConfigsvrSize` value to 4 or I should change the arbiter checks logic?

Reply:
@delgod FYI